tests/sentry/issue_detection/test_large_http_payload_detector.py

```diff
@@ -305,63 +305,30 @@ def test_does_not_trigger_detection_for_prefetch_spans(self) -> None:
         assert len(self.find_problems(event)) == 0
 
     @with_feature("organizations:large-http-payload-detector-improvements")
-    def test_does_not_trigger_detection_for_file_downloads_with_content_disposition_attachment(
-        self,
-    ) -> None:
-        """Test that spans with Content-Disposition: attachment are excluded"""
-        settings = get_detection_settings(organization=self.organization)
-
+    def test_does_not_trigger_detection_for_filtered_paths(self) -> None:
+        project = self.create_project()
+        ProjectOption.objects.set_value(
+            project=project,
+            key="sentry:performance_issue_settings",
+            value={"large_http_payload_filtered_paths": "/api/0/organizations/download/"},
+        )
+        settings = get_detection_settings(project.id, organization=self.organization)
         spans = [
             create_span(
                 "http.client",
-                hash="hash1",
-                desc="GET /api/0/organizations/download/file/12345",
-                duration=1000.0,
+                1000,
+                "GET /api/0/organizations/download/endpoint1",
```

Comment:
i know that partial matches are supported (e.g. `/api/` matches `/api/users`), but is it intentional that something like `/api/user` will also match `/api/users`?